src/Tribe/Views/V2/View.php

```diff
@@ -1765,28 +1765,23 @@ protected function on_page_reset() {
 	protected function get_breadcrumbs() {
 		$context     = $this->context;
 		$breadcrumbs = [];
-		$context_tax = $context->get( TEC::TAXONOMY, false );
+		$taxonomy    = TEC::TAXONOMY;
+		$context_tax = $context->get( $taxonomy, false );
 
 		// Get term slug if taxonomy is not empty
 		if ( ! empty( $context_tax ) ) {
-			$taxonomy  = $context->get( 'taxonomy', false );
-			$term_slug = $taxonomy ? $context->get( $taxonomy, false ) : false;
-
-			// Set up breadcrumbs for category
-			if ( ! empty( $term_slug ) ) {
-				$term  = get_term_by( 'slug', $term_slug, $taxonomy );
-				if ( ! empty( $term->name ) ) {
-					$label = $term->name;
-
-					$breadcrumbs[] = [
-						'link'  => $this->get_today_url( true ),
-						'label' => tribe_get_event_label_plural(),
-					];
-					$breadcrumbs[] = [
-						'link'  => '',
-						'label' => $label,
-					];
-				}
+			$term  = get_term_by( 'slug', $context_tax, $taxonomy );
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :warning: Equals sign not aligned correctly; expected 1 space but found 2 spaces

<sub>hash: 1fa32fc21b847fe72a30ec1b7122d4ed</sub>